lib/_stream_writable.js

```diff
@@ -325,6 +325,12 @@ Writable.prototype.setDefaultEncoding = function setDefaultEncoding(encoding) {
   return this;
 };
 
+Object.defineProperty(Writable.prototype, 'writableBuffer', {
+  get: function() {
```

Comment:
does this need to be `enumerable: false` like the others?

Reply:
Yes my bad.